SEN381_API_GROUP3/SEN381_API_GROUP3/Controllers/MedicalConditionController.cs

```diff
@@ -29,20 +29,23 @@ public List<MedicalCondition> Get(int id)
 
         // POST api/<MedicalConditionController>
         [HttpPost]
-        public void Post([FromBody] string value)
+        public void Post(string MedicalConditionName, string MedicalConditionDescription, int TreatmentID)
```

Comment:
Pasop om methods name te gee soos Post, of Update, of Delete. Vir nou is dit fine, maar kan lei na ambiguities as daat meer as een post request op een controller kom. gebruik eeder name soos public void getMedicalCondition. Otherwise, code looks great!